package.json

```
@@ -4,7 +4,7 @@
   "description": "Core Components Library Developed & Maintained By The Articles and Reach & Languages Team",
   "main": "index.js",
   "scripts": {
-    "build": "lerna exec --parallel -- babel src -d dist --ignore spec.js,story.js",
+    "build": "lerna exec --parallel -- babel src -d dist --ignore test.js,stories.js",
```

Our test and story files are JSX, not JS:

```suggestion
    "build": "lerna exec --parallel -- babel src -d dist --ignore test.jsx,stories.jsx",
```